### DISCUSSION TITLE
Default alignment

### DISCUSSION THREAD

**Discussion by CraigHollaman** [voidlabs/mosaico#679] [Q&A] [answered]
How do I make the text block be Centered as a default. 
If I change this it complains.... I just need it to be centered as the first option. 
**text-align: left;**
 line-height: normal; -ko-font-size: @[longTextStyle.size]px; -ko-color: @longTextStyle.color; -ko-font-family: @longTextStyle.face; -ko-text-align: @longTextStyle.align; -ko-attr-align: @longTextStyle.align; -ko-line-height: @[longTextStyle.lineHeight]" align="left" data-ko-editable="longText">
Thanks.

**Comment by bago** [comment, marked as answer]
Please note that styles shared between blocks in the same "theme" have to be consistents.
So if you change longTextStyle.align to center you have to change it everywhere the same style is used in other blocks sharing the same theme.

If you want a block to not follow the same theme of other blocks then delcare it in a new theme.

In versafix you can see there is a frameTheme and a contentTheme and you can see some block belongs to the frameTheme while others belong to the contentTheme.

The longTextStyle.color property must be the same for every block in the contentTheme but it can be different from the color used in the frameTheme's blocks.

This is needed in order to correctly use the global/local changes when working with the template.

**Comment by CraigHollaman** [comment]
How do I change the contentTheme ?

**Comment by bago** [comment]
contentTheme.longTextStyle.align is automatically generated the first time longTextStyle.align is used in a contentTheme block. The following occourences will check if the value is the same of the theme and complaint otherwise (as you found out).

So, if you simply change the text-align: from left to center in every occourence of the "longTextStyle.align" in blocks using contentTheme block (or everywhere if you want to change it also in the frameTheme).

If you are working in the MML source ( https://github.com/voidlabs/versafix-template/blob/master/template-def/template-versafix-1.htmml#L977 ) for versafix then you simply have to change:
```
  <default selector="text, title-text, long-text:not([fixed-align]), btn, linked-text" align="left"></default>
```
with
```
  <default selector="text, title-text, btn, linked-text" align="left"></default>
  <default selector="long-text:not([fixed-align])" align="center"></default>
```

**Reply by CraigHollaman** [reply]
Great. Thanks for your help.
